posthog/api/query.py

```diff
@@ -102,6 +106,13 @@ class QueryViewSet(TeamAndOrgViewSetMixin, PydanticModelMixin, viewsets.ViewSet)
     def get_throttles(self):
         if self.action == "draft_sql":
             return [AIBurstRateThrottle(), AISustainedRateThrottle()]
+        if (
+            settings.API_QUERIES_ENABLED
+            and self.team
+            and self.team.organization.is_feature_available(AvailableFeature.API_QUERIES_CONCURRENCY)
```

Comment:
@zlwaterfield if i want to enable this only to selected orgs, i must call is_feature_available (i understand we add only selected orgs to a plan with this feature available).

This will be an extra call to db for many orgs, suggestions how to cache it in redis?

Reply:
OK, i've added a proper cache